daisy_workflows/image_import/import_image.sh

```diff
@@ -43,6 +43,27 @@ echo "DISKNAME: ${DISKNAME}" 2> /dev/null
 echo "ME: ${ME}" 2> /dev/null
 echo "ZONE: ${ZONE}" 2> /dev/null
 
+function waitUntilDeviceAttached() {
+  local devPath="${1}"
+  local capacityGb="${2}"
+
+  echo "Import: Checking for $devPath ${capacityGb}G"
+  for t in {1..60}; do
+    if [[ -e ${devPath} ]]; then
+      capacity=$(lsblk ${devPath})
+      if [[ "${capacity}" =~ "${capacityGb}G" ]]; then
+          echo "Import: $devPath is attached and ready."
```

Comment:
Seems indentation is incorrect here (return and echo are in the same block)

Reply:
good catch!